packages/components/src/components/hds/dropdown/list-item/interactive.hbs

```diff
@@ -30,7 +34,11 @@
         </span>
       {{/if}}
       <Hds::Text::Body class="hds-dropdown-list-item__interactive-text" @tag="span" @size="200" @weight="medium">
-        {{this.text}}
+        {{#if (has-block)}}
+          {{yield (hash Badge=(component "hds/badge" size="small"))}}
```

Comment:
question: Is it intentional that we enforce the size here, but not [on the `@isLoading` variant](https://github.com/hashicorp/design-system/pull/2346/files#diff-6fc55f901fc63dbb36f21d330801f0ebd9b7efc61f2a2fb104893b72c9484323R13)?

Reply:
No, it's not. Thanks. Probably need to add some variations for that in showcase too.

Reply:
In the Dropdown Toggle Button they are always `small`, same for the Checkbox, Checkmark, and Radio equivalents. The main consideration for that is to ensure the resulting height of the ListItem is always the same (a large Badge would likely increase it). Maybe check with @LilithJames-HDS (if this hasn't already been discussed).